.changeset/seven-flies-chew.md

```diff
@@ -0,0 +1,11 @@
+---
+"@latticexyz/services": major
+"@latticexyz/network": major
```

Comment:
is `network` being deleted a major change? should we just omit this here so it doesn't try to release a thing that no longer exists?

Reply:
yeah i assume changeset will error if we mention network here after it was removed
```suggestion
```